ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py

```diff
@@ -53,3 +53,37 @@ def test_numpy_argsort(
         x=np.asarray(x, dtype=input_dtype),
         axis=axis,
     )
+
+
+# argmax
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.statistical_dtype_values(function="argmax"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.argmax"
+    ),
+)
+def test_numpy_argmax(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x, axis = dtype_and_x
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="argmax",
+        input=np.asarray(x, dtype=input_dtype),
```

Comment:
These keyword parameters should match the names of the frontend method. Thus, `input` should be renamed to `a`.